api/src/pcapi/templates/extracts/beneficiary_extract.html

```diff
@@ -0,0 +1,194 @@
+        </table>
+
+        <h3>🌐 <span class=underline>Historique des moyens de connexion</span></h3>
+        {% for device in container.internal.loginDevices%}
+            <table class="borderless">
+                <tr><td>Date de première connexion</td><td>{{device.dateCreated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                <tr><td>Identifiant de l'appareil</td><td>{{device.deviceId}}</td></tr>
+                <tr><td>Type d'appareil</td><td>{{device.source}}</td></tr>
+                <tr><td>Système d'exploitation</td><td>{{device.os}}</td></tr>
+            </table>
+        {% endfor %}
+
+        <h3>📧 <span class=underline>Historique des changements d'adresse de messagerie</span></h3>
+        {% for history in container.internal.emailsHistory %}
+            <table class="borderless">
+                <tr><td>Date de la demande</td><td>{{history.dateCreated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                <tr><td>Ancienne adresse email</td><td>{{history.oldEmail}}</td></tr>
+                <tr><td>Nouvelle adresse email</td><td>{{history.newEmail}}</td></tr>
+            </table>
+        {% endfor %}
+
+        {% if container.internal.actionsHistory %}
+            <h3>⛔ <span class=underline>Historique des bloquages du compte « pass Culture »</span></h3>
+                {% for action in container.internal.actionsHistory %}
+                    <table class="borderless">
+                        <tr><td>Date</td><td>{{action.actionDate.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                        <tr><td>Action</td><td>{{action.actionType.value}}</td></tr>
+                    </table>
+                {% endfor %}
+        {% endif %}
+
+        <h3>👌 <span class=underline>Validations d'identité</span></h3>
+        {% for validation in container.internal.beneficiaryValidations %}
+            <table class="borderless">
+                <tr><td>Date de la validation</td><td>{{validation.dateCreated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                <tr><td>Moyen de la validation</td><td>{{validation.type.value}}</td></tr>
+                {% if validation.status %}<tr><td>Résultat</td><td>{{validation.status.value}}</td></tr>{%endif%}
+                {% if validation.updateAt %} <tr><td>Derniere modification</td><td>{{validation.updatedAt.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>{%endif%}
+            </table>
+        {% endfor %}
+
+        <h3>💰 <span class=underline>Crédits</span></h3>
+        {% for deposit in container.internal.deposits%}
+            <table class="borderless">
+                <tr><td>Date d'obtention</td><td>{{deposit.dateCreated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                {% if deposit.expirationDate %}<tr><td>Date d'expiration</td><td>{{deposit.expirationDate.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>{% endif %}
+                <tr><td>Valeur</td><td>{{'%0.2f' % deposit.amount|float}}€</td></tr>
```

Comment:
Je pressens un point au lieu d'une virgule dans le montant.
Suggestion non testée : 
```suggestion
                <tr><td>Valeur</td><td>{{('%0.2f' % deposit.amount|float) | replace('.', ',') }} €</td></tr>
```
Ça peut concerner d'autres montants plus bas.